.dependency-cruiser.js

```diff
@@ -28,7 +71,9 @@ module.exports = {
           '(^|/)\\.[^/]+\\.(js|cjs|mjs|ts|json)$', // dot files
           '\\.d\\.ts$',                            // TypeScript declaration files
           '(^|/)tsconfig\\.json$',                 // TypeScript config
-          '(^|/)(babel|webpack)\\.config\\.(js|cjs|mjs|ts|json)$' // other configs
+          '(^|/)(babel|webpack)\\.config\\.(js|cjs|mjs|ts|json)$', // other configs
+          dir('src/vscode/webview'), // build output
```

Comment:
Could we fetch this and some other entries from .gitignore instead?

Reply:
In principle we could. But since dependency cruiser doesn't support this out of the box, it would be significantly more effort than manually adding all `.gitignore` entries to `.dependency-cruiser.js`.